### DISCUSSION TITLE
Name change

### DISCUSSION THREAD

**Discussion by rae-hill** [jacob-alford/schemata-ts#127] [Ideas]
I've thought about changing the name to `schema-ts` (there's no way that's available) or `schema-ts-types`.  

Not sure how hard it will be to update in NPM - changing the name.  

Thoughts?

**Comment by skeate** [comment]
there is no package on npmjs named schema-ts

I don't think it's a problem to change names; just publish with the new name in package.json. And manually remove the old one, if you want.

**Reply by rae-hill** [reply]
Cool, I will dwell on it for a bit.  

NPM also has restrictions on _similar_ names, but I think `schema-ts` would be an awesome name
